src/qt/bitcoingui.cpp

```diff
@@ -400,8 +400,8 @@ void BitcoinGUI::createActions()
 
     openInfoAction = new QAction(tr("&Information"), this);
     openInfoAction->setStatusTip(tr("Show diagnostic information"));
-    openRPCConsoleAction = new QAction(tr("&Debug console"), this);
-    openRPCConsoleAction->setStatusTip(tr("Open debugging console"));
+    openRPCConsoleAction = new QAction(tr("&Tools window"), this);
```

Comment:
17096: (unlike in bitcoin) we have menu items for each tab here, so this change is not applicable imo
```suggestion
    openRPCConsoleAction = new QAction(tr("&Debug console"), this);
```